tfgrid3/ubuntu24.04/fullvm/ubuntu24-fullvm-flist-builder.sh

```diff
@@ -55,11 +55,11 @@ if ! command -v extract-vmlinux &>/dev/null; then
 fi
 
 # Kernel Extraction
-extract-vmlinux ubuntu-jammy/boot/vmlinuz | tee ubuntu-jammy/boot/vmlinuz-5.15.0-25-generic.elf > /dev/null
-mv ubuntu-jammy/boot/vmlinuz-5.15.0-25-generic.elf ubuntu-jammy/boot/vmlinuz-5.15.0-25-generic
+extract-vmlinux ubuntu-noble/boot/vmlinuz | tee ubuntu-noble/boot/vmlinuz-6.8.0-31-generic.elf > /dev/null
+mv ubuntu-noble/boot/vmlinuz-6.8.0-31-generic.elf ubuntu-noble/boot/vmlinuz-6.8.0-31-generic
 
 # Create a compressed archive of the configured system for uploading to hub.
-tar -czf ubuntu-jammy.tar.gz -C ubuntu-jammy .
+tar -czf ubuntu-noble.tar.gz -C ubuntu-noble .
```

Comment:
could you please use the version number in the name of the flist better than using image codename to be more clear for users when adding it to hub then dashboard

Reply:
- updated